contracts/scripts/app.ts

```diff
@@ -43,6 +63,26 @@ async function main(): Promise<void> {
   const registry = await (await GrantRegistryFactory.deploy()).deployed();
   console.log(`Deployed GrantRegistry to ${registry.address}`);
 
+  const GrantRoundFactory: ContractFactory = await ethers.getContractFactory('GrantRoundFactory', deployer);
+  const roundFactory = await (await GrantRoundFactory.deploy()).deployed();
+  console.log(`Deployed GrantRoundFactory to ${roundFactory.address}`);
+
+  // create the rounds
+  await Promise.all(
+    rounds.map((round) =>
+      roundFactory.createGrantRound(
```

Comment:
Looks like a rebase may have broken this. When I go to run `yarn dev`, it errors here because we're missing the `payoutAdmin` argument for deployment.

Additionally, after modifying this call to include the missing parameter, the UI starts, but does not show any Rounds at `/dgrants/rounds/`:

Is the app making an assumption about the address of the factory? Do I need to use a specific mnemonic to match that dpeloyment address, for example?

Reply:
Related: `owner: round.owner()` on L101 of `data.ts` is also broken. We now have two separate privileged roles so that should be changed to:

```javascript
payoutAdmin: await round.payoutAdmin(),
metadataAdmin: await round.metadataAdmin()
```

Reply:
Yes, I think that's correct @mds1. When I added an extra param, it worked, but did not show Rounds in the UI, which is the second part of my comment.

Reply:
@apbendi - The name change will likely have caused the rpc calls to fail - all fixed up and working as expected in b35f417 🙌